include/swift/AST/DiagnosticsDriver.def

```diff
@@ -183,23 +183,28 @@ WARNING(warn_use_filelists_deprecated, none,
         "'-driver-filelist-threshold=0' instead", ())
 
 WARNING(warn_unable_to_load_swift_ranges, none,
-"unable to load swift ranges file \"%0\", %1",
-(StringRef, StringRef))
+       "unable to load swift ranges file \"%0\", %1",
+       (StringRef, StringRef))
 
 WARNING(warn_unable_to_load_compiled_swift, none,
-"unable to load previously compiled swift file \"%0\", %1",
-(StringRef, StringRef))
+       "unable to load previously compiled swift file \"%0\", %1",
+       (StringRef, StringRef))
 
 WARNING(warn_unable_to_load_primary, none,
-"unable to load primary swift file \"%0\", %1",
-(StringRef, StringRef))
+       "unable to load primary swift file \"%0\", %1",
+       (StringRef, StringRef))
```

Comment:
Nit these should all be aligned one more column over with the selector name, not the open paren.